qiskit/qpy/formats.py

```diff
@@ -173,6 +173,14 @@
 SPARSE_PAULI_OP_LIST_ELEM_PACK = "!Q"
 SPARSE_PAULI_OP_LIST_ELEM_SIZE = struct.calcsize(SPARSE_PAULI_OP_LIST_ELEM_PACK)
 
+# SparseObservable List - Version 17+
+SPARSE_OBSERVABLE_OP_LIST_ELEM = namedtuple(
```

Comment:
Could we call this `SPARSE_OBSERVABLE`? Not quite sure why the original thing had the `LIST_ELEM` suffix, this is just a single `SparseObservable`